Bayesian_Cognitive_Modeling/ParameterEstimation/DataAnalysis/Correlation_2_Stan.R

```diff
@@ -72,7 +30,7 @@ parameters <- c("r", "mu", "sigma")
 
 # The following command calls Stan with specific options.
 # For a detailed description type "?rstan".
-samples <- stan(model_code=model,   
+samples <- stan(file="Correlation_2.stan",   
```

Comment:
Thanks---this is a huge improvement putting them in files.  I'd use `correlation-2.stan`.  File names conventionally have hyphens rather than underscores (unless we're in C++ or Python packages).  But if you can, an even more informative and not so long name would be useful like `meas-err.stan`.